releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -4,6 +4,10 @@ The Azure App Configuration extension in Azure DevOps pipeline has been publishe
 
 ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v6.0.0 - February, 02 2023
+* Upgraded task to use Node 16, previously used Node 10.
```

Comment:
"Node", should that be "Node.js" ? 

Also, I'd add a bit to flush the sentence out. "Updated the task to require Node.js 16. It previously required version 10."

Reply:
Updated statement to "Updated the task to requires Node.js 16. It previously required 10"